src/components/application_manager/include/application_manager/smart_object_keys.h

```diff
@@ -216,6 +216,7 @@ extern const char* key_press_mode;
 // duplicate names from hmi_request
 extern const char* limited_character_list;
 extern const char* auto_complete_text;
+extern const char* auto_complete_list;
```

Comment:
@LuxoftAKutsan, @dtrunov Fixed in b10f1c6, also added TODO to check all other duplicate fields and leave only one exemplar for each of those fealds.